packages/fuel-indexer-api-server/Cargo.toml

```diff
@@ -27,7 +27,7 @@ prometheus = { version = "0.13", optional = true }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0", features = ["raw_value"] }
 serde_yaml = "0.8"
-sqlx = { version = "0.6", features = ["postgres", "runtime-tokio-rustls", "sqlite"] }
+sqlx = { version = "0.6", features = ["postgres", "runtime-tokio-rustls", "sqlite", "bigdecimal"] }
```

Comment:
We can remove the `sqlite` feature here.

Reply:
@deekerno True. I saw that you took care of that in https://github.com/FuelLabs/fuel-indexer/pull/563 so figured we'll just do that when your PR is merged